src/gluonts/dataset/repository/_m3.py

```diff
@@ -192,10 +196,12 @@ def normalize_category(c: str):
             )
         )
 
-    train_file = dataset_path / "train" / "data.json"
-    test_file = dataset_path / "test" / "data.json"
+    train_path = dataset_path / "train"
+    test_path = dataset_path / "test"
+    train_path.mkdir(exist_ok=True)
+    test_path.mkdir(exist_ok=True)
```

Comment:
Since we do this over and over again, maybe we can create a utility for this.

Maybe something like this:

```py
def create_dataset_paths(base_path: Path, channel_names: List[str]):
    base_path.mkdir(exists_ok)

    channels = {}

    for channel_name in channel_names:
        channel = base_path / channel_name
        channel.mkdir(exists_ok=True)

        channels[channel_name] = channel

    return channels



paths = create_dataset_paths(path, ["train", "test"])
dataset_writer.write_to_folder(train_data, paths["train"])
dataset_writer.write_to_folder(test_data, paths["test"])
```

Reply:
Ah, it looks like we already have this in `TrainDatasets`.